feat: LEAP-1782: add pauses to members list

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13276854332)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/412
<!-- Follow Merge -->
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/412 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13269291036)

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13269291036

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13276682824)
> Successfully merged:  24 files changed, 921 insertions(+), 3 deletions(-)

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13276673584

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13276720558)
> Successfully merged:  24 files changed, 921 insertions(+), 3 deletions(-)

jombooth committed:
Merge branch 'develop' into 'fb-LEAP-1782'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13276720558

jombooth commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13276854332)

jombooth committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13276854332
